lib/lyber_core/rspec.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module LyberCore
+  # Methods to support testing robots
+  module Rspec
```

Comment:
interesting.   Makes me ponder if this should be made avail downstream in aspec/support folder (where it really belongs) ...